setup.py

```diff
@@ -11,9 +12,15 @@
     "pytest-cov==3.*",
     "pytest-vcr==1.*",
     "pytest==7.*",
+    "types-requests",
+    "types-urllib3",
     "vcrpy==4.*",
 ]
 
+CPYTHON_DEV_REQUIREMENTS = [
```

Comment:
I hate this. At the earliest convenience (once we can confirm nobody is using `pypy`, let's remove this weirdness.

Can you leave an inline comment as to why these are broken out the way they are?